tools/driver/cling.cpp

```diff
@@ -106,6 +107,29 @@ int main( int argc, char **argv ) {
     Interp.loadFile(Lib);
 
   cling::UserInterface Ui(Interp);
+
+  // Get the home directory
+  std::string HomeDir;
+#ifdef __unix__
+  HomeDir = getenv("HOME");
+  HomeDir += "/";
+#elif defined(_WIN32)
+  HomeDir = getenv("USERPROFILE");
+  HomeDir += "\\";
+#endif
```

Comment:
There's [`llvm::sys::home_directory()`](https://llvm.org/doxygen/namespacellvm_1_1sys_1_1path.html#a1286bcdea03c97a18eb1f41af524f3d3) for that.